examples/.tailcallrc.graphql

```diff
@@ -22,35 +123,312 @@ enum HttpVersion {
+  query: Query
+}
+
+type User {
+  id: Int!
+  name: String!
+  username: String!
+  email: String!
+  phone: String
+  website: String
+  street: String
+}
+
+type Query {
+  user(id: Int): Post!
+}
+```
+
+In the example, `@modify(omit: true)` on `address` removes the `Address` type from the schema.
+
+`@addField` handles field nullability. If a path field is nullable, the result is nullable.
+
+`@addField` also supports indexing. For a `posts` field of type `[Post]`, to get the first post's title, specify the path as [`"posts"`,`"0"`,`"title"`].
+
+```graphql
+type User @addField(name: "firstPostTitle", path: ["posts", "0", "title"]) {
+  id: Int!
+  name: String!
+  username: String!
+  email: String!
+  phone: String
+  website: String
+  posts: Post @http(path: "/users/{{value.id}}/posts")
+}
+
+type Post {
+  id: Int!
+  userId: Int!
+  title: String!
+  body: String!
+}
+```
+"""
 directive @addField(name: String, path: [String]!) on OBJECT
+"""
+The `@modify` operator in GraphQL allows you to change field or node attributes in your schema. Here's how:
+
+### name
+
+The `name` argument in `@modify` lets you rename a field or node. This is useful when the field name in your data source doesn't match your schema's desired field name. Example:
+
+```graphql
+type User {
+  id: Int! @modify(name: "userId")
+}
+```
+
+Here, `@modify(name: "userId")` instructs GraphQL to present the field as `userId` in your schema, even though it's referred to as `id` in the data source.
+
+### omit
+
+The `omit` argument in `@modify` lets you exclude a field or node from your schema. This is useful for hiding certain data from the client. Example:
+
+```graphql
+type User {
+  id: Int! @modify(omit: true)
+}
+```
+
+Here, `@modify(omit: true)` instructs GraphQL not to include the `id` field in the schema, making it inaccessible to the client.
+"""
 directive @modify(omit: Boolean, name: String) on FIELD_DEFINITION
+
+"""
+The `groupBy` parameter groups multiple data requests into a single call. For more details please refer out [n + 1 guide](https://tailcall.run/docs/guides/n+1#solving-using-batching).
+
+```graphql
+type Post {
+  id: Int!
+  name: String!
+  user: User @http(path: "/users", query: [{key: "id", value: "{{value.userId}}"}], groupBy: ["id"])
+}
+```
+
+- `query: {key: "id", value: "{{value.userId}}"}]`: Here, TailCall CLI is instructed to generate a URL where the user id aligns with the `userId` from the parent `Post`. For a batch of posts, the CLI compiles a single URL, such as `/users?id=1&id=2&id=3...id=10`, consolidating multiple requests into one.
+"""
 directive @groupBy(path: [String!]) on FIELD_DEFINITION
+"""
+The `@const` operators allows us to embed a constant response for the schema. For eg:
+
+```graphql
+schema {
+  query: Query
+}
+
+type Query {
+  user: User @const(data: {name: "John", age: 12})
+}
+
+type User {
+  name: String
+  age: Int
+}
+```
+"""
 directive @const(data: JSON) on FIELD_DEFINITION
 directive @graphQL(
```

Comment:
I guess we need docs also for these (and it's respective fields)

Reply:
Agree.